src/sentry/services/hybrid_cloud/organization_mapping/model.py

```diff
@@ -10,13 +10,10 @@
 from pydantic.fields import Field
 from typing_extensions import TypedDict
 
-from sentry.services.hybrid_cloud import RpcModel
+from sentry.services.hybrid_cloud.organization import RpcOrganizationSummary
 
 
-class RpcOrganizationMapping(RpcModel):
-    organization_id: int = -1
```

Comment:
Note that this is a potentially breaking change: `organization_id` is renamed to `id` with the same meaning. I didn't find any places that `organization_id` was referenced, however. (We'll see if CI turns one up.)

I don't love that `id` could be confused for the primary key of the `OrganizationMapping` table (though we weren't exposing that before and shouldn't start). I think it's worth it to be able to up-cast `RpcOrganizationMapping` to `RpcOrganizationSummary`. We should be encouraging the caller to think of both classes as a miniature slice of an `RpcOrganization`, so `id` should mean the org's ID in all contexts.

Reply:
That seems fair.

Reply:
> We should be encouraging the caller to think of both classes as a miniature slice of an RpcOrganization, so id should mean the org's ID in all contexts.

I agree here. The id of a mapping row isn't something we should need to reference directly, especially once `organization_id` becomes a unique key on the mapping table.